.github/workflows/build-test.yml

```diff
@@ -14,194 +14,194 @@ jobs:
+#          mvn verify -pl :gremlin-console -DskipTests -DskipIntegrationTests=false
+#  javascript:
+#    name: javascript
+#    timeout-minutes: 15
+#    needs: smoke
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set up JDK 11
+#        uses: actions/setup-java@v2
+#        with:
+#          java-version: '11'
+#          distribution: 'temurin'
+#      - name: Build with Maven
+#        run: |
+#          mvn clean install -pl -:gremlin-python,-gremlin-dotnet,-:gremlin-dotnet-source,-:gremlin-dotnet-tests -q -DskipTests -Dci
+#          mvn verify -pl :gremlin-javascript,:gremlint -DincludeNeo4j
+#  python:
+#    name: python
+#    timeout-minutes: 20
+#    needs: smoke
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set up JDK 11
+#        uses: actions/setup-java@v2
+#        with:
+#          java-version: '11'
+#          distribution: 'temurin'
+#      - name: Set up Python 3.x
+#        uses: actions/setup-python@v2
+#        with:
+#          python-version: '3.8'
+#      - name: Setup Python requirements
+#        run: |
+#          sudo apt install gcc libkrb5-dev
+#          python3 -m pip install --upgrade pip
+#          pip install virtualenv
+#      - name: Build with Maven
+#        run: |
+#          touch gremlin-python/.glv
+#          mvn clean install -pl -:gremlin-javascript,-gremlin-dotnet,-:gremlin-dotnet-source,-:gremlin-dotnet-tests,-:gremlint -q -DskipTests -Dci
+#          mvn verify -pl gremlin-python -DincludeNeo4j
+#  dotnet:
+#    name: .NET
+#    timeout-minutes: 20
+#    needs: smoke
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set up JDK11
+#        uses: actions/setup-java@v2
+#        with:
+#          java-version: '11'
+#          distribution: 'temurin'
+#      - name: Set up .NET 6.0.x
+#        uses: actions/setup-dotnet@v1
+#        with:
+#          dotnet-version: '6.0.x'
+#      - name: Build with Maven
+#        run: |
+#          touch gremlin-dotnet/src/.glv
+#          touch gremlin-dotnet/test/.glv
+#          mvn clean install -pl -:gremlin-javascript,-:gremlin-python,-:gremlint -q -DskipTests -Dci
+#          mvn verify -pl :gremlin-dotnet,:gremlin-dotnet-tests -P gremlin-dotnet -DincludeNeo4j
+#  neo4j-gremlin:
+#    name: neo4j-gremlin
+#    timeout-minutes: 20
+#    needs: smoke
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set up JDK11
+#        uses: actions/setup-java@v2
+#        with:
+#          java-version: '11'
+#          distribution: 'temurin'
+#      - name: Build with Maven
+#        run: |
+#          mvn clean install -pl -:gremlin-javascript,-:gremlin-python,-gremlin-dotnet,-:gremlin-dotnet-source,-:gremlin-dotnet-tests -q -DskipTests -Dci
+#          mvn verify -pl :neo4j-gremlin -DincludeNeo4j
```

Comment:
Should we just remove them instead of commenting?

Reply:
Maybe I should. It'll look cleaner for now.